specification/deviceprovisioningservices/resource-manager/readme.java.md

```diff
@@ -0,0 +1,34 @@
+# Java DPS
+
+> see <https://aka.ms/autorest>
+
+This is the AutoRest configuration file for the Device Provisioning Service.
+
+## Java common settings
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to "SDKs" directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(java)
+java:
+  azure-arm: true
+  regenerate-manager: true
+  license-header: MICROSOFT_MIT_NO_VERSION
```

Comment:
Elsewhere, I see `MICROSOFT_MIT_NO_CODEGEN` instead. Which should I specify?

Reply:
For track1, it would be `MICROSOFT_MIT_NO_CODEGEN`

For track2, it should be `MICROSOFT_MIT_SMALL`, but you can omit it (and many other configure) as pipeline will provide the defaults and override the impropriate configures.
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/mgmt/automation/parameters.py#L31